### ISSUE TITLE
[Poe2 feedback] - map assets stretching out of the box.

### ISSUE THREAD

**Issue by zubSero** [RoseTheFlower/UltrawideIndex#103]
Fix works flawlessly. 
So far the only thing that looks fubar is the world map.
This page starts heavily to flicker and the borders are overlapping with the static asset behind it.

I can not really mark it as a bug as I am not sure if its GGG messing up or perhaps the widescreen fix itself.
Just wanted to give a headsup,

**Comment by RoseTheFlower** [comment]
If you're using the fix for the first game, see #100.

**Comment by zubSero** [comment]
Perhaps i am going crazy, i was using the newer released fix specifically for poe2 which i downloaded this morning. 
I still experienced the assets stretch and flickering, but while i created this issue I rebooted my entire pc to allow Nvidia drivers to be updated. I disabled Nvidia reflex as well as i had it on the boost option

Flickering is gone now. the assets poking out are not really fixable i guess thats just the way its coded. 
il just mark it as closed seems like an user issue.
